src/tests/ftest/container/container_async.py

```diff
@@ -72,30 +70,21 @@ def test_createasync(self):
         global GLOB_SIGNAL
         global GLOB_RC
 
-        # parameters used in pool create
-        createmode = self.params.get("mode", '/run/createtests/createmode/*/')
-        createsetid = self.params.get("setname", '/run/createtests/createset/')
-        createsize = self.params.get("size", '/run/createtests/createsize/')
-        createuid = os.geteuid()
-        creategid = os.getegid()
-
-        try:
-            # initialize a python pool object then create the underlying
-            # daos storage
-            self.pool = DaosPool(self.context)
-
-            self.pool.create(createmode, createuid, creategid,
-                             createsize, createsetid, None)
+        # initialize a python pool object then create the underlying
+        # daos storage
+        self.pool = TestPool(
```

Comment:
Given that these 4 lines of code (`self.pool = TestPool(...)`, `self.pool.get_params(self)`, `self.pool.create()`, and `self.pool.connect()`) are repeated in this file (and the other files being modified), maybe we should add this as a method to the `TestWithServers` class or a common utils base test class.